libnavigation-util/src/main/java/com/mapbox/navigation/utils/audio/AudioTypeChain.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.navigation.core.telemetry.audio
```

Comment:
> Package directive doesn't match file location

Same thing applies to other classes 👇

Reply:
👍

Reply:
This https://github.com/mapbox/mapbox-navigation-android/pull/2445/files#r380967783 still applies.

Reply:
👍 removed